mock/mock.spec

```diff
@@ -9,7 +9,7 @@
 
 Summary: Builds packages inside chroots
 Name: mock
-Version: 4.1
+Version: 4.1.post
```

Comment:
???

Reply:
This is an experiment (likely I should use `post1`) that I don't insist on,
but seems really attractive.

Simply, this way I can signalize that the Version is higher than the one
released in Fedora, always beating the NVR there (even if Release goes up due
to rebuilds) and in mock-core-configs I still can require the version newer
then the one from this PR.

Reply:
I moved this experiment into #1138, it was rather disturbing change in this PR.